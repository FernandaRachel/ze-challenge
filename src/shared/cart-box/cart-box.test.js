import React from 'react';
import renderer from 'react-test-renderer';
import CartBox from './cart-box'

describe('CarBox', () => {
    test('snapshot renders', () => {
        const arr = [];
        const component = renderer.create(<CartBox cartList={arr} />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

