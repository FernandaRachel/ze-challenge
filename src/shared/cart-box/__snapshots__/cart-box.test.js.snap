// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarBox snapshot renders 1`] = `""`;
